OpenFilesScreen.c

```diff
@@ -115,7 +118,7 @@ static OpenFiles_ProcessData* OpenFilesScreen_getProcessData(pid_t pid) {
       close(fdnull);
       char buffer[32] = {0};
       xSnprintf(buffer, sizeof(buffer), "%d", pid);
-      execlp("lsof", "lsof", "-P", "-p", buffer, "-F", NULL);
+      execlp("lsof", "lsof", "-P", "-o", "-p", buffer, "-F", NULL);
```

Comment:
This is <s>broken</s> FUBAR in `lsof`:
[TW: Atrocious code, click at your own risk! Cannot be unseen!](https://github.com/lsof-org/lsof/blob/234ac9ca6082084640cb1b762f6721eccc993c79/print.c#L1003-L1074)